src/mahoji/commands/clue.ts

```diff
@@ -396,7 +434,7 @@ export const clueCommand: OSBMahojiCommand = {
 
 		response.content = `${user.minionName} is now completing ${quantity}x ${
 			clueTier.name
-		} clues, it'll take around ${formatDuration(duration)} to finish.${
+		} clues, it'll take around ${formatDuration(duration)} to finish (${((quantity / duration) * 3600000).toFixed(1)}/hr).${
```

Comment:
Dont use magic numbers (36000000), in this case I think you want to use `Time.Hour`.